### PR TITLE
feat(blog): semantic search blog

### PR DESCRIPTION
**Pull request by rachlllg**
Add a new blog post for semantic search finetuning and inception

issue: https://github.com/freelawproject/courtlistener/issues/5166

### CODE REVIEW

**Review by mlissner — CHANGES_REQUESTED** [review]
I think it's good. I made some tweaks to it, and committed to your branch. One thing I didn't do is explain what a loss function is, but I left a comment in your post (look for my initials, MJL).

If you can fix that, I think we are ready to post this.

Thank you!

**Review by mlissner — APPROVED** [review]
I made a few more small tweaks, but this is great. Launching!
